### PR TITLE
Ajna borrow and earn history

### PR DESCRIPTION
**Pull request by piekczyk**
# [UI components](https://app.shortcut.com/oazo-apps/story/9553/ajna-history-ui-components)
# [Datasorce](https://app.shortcut.com/oazo-apps/story/9552/ajna-history-map-front-end-to-subgraph-events)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added set of components to handle history
- addded mappers to handle new history event interface
- handled borrow and earn ajna history
  
## How to test 🧪
  <Please explain how to test your changes>

- on relatively new Ajna position go to history tab and verify whether everything looks ok (except for LTV and fees due to lack of tokens prices on goerli)

### CODE REVIEW

File: components/history/PositionHistory.tsx [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+  etherscanUrl,
+  collateralToken,
+  quoteToken,
+  isShort,
+  priceFormat,
+}) => {
+  const [opened, setOpened] = useState(false)
+  const { t } = useTranslation()
+
+  const humanDate = new Date(item.timestamp!).toLocaleDateString('en-US', {
```
Comment (piotrkonowrocki) [inline components/history/PositionHistory.tsx:184]: As the month is displayed in textural form, I think language should come from settings, like it is in `timeAgo` function.

File: components/history/PositionHistory.tsx
[the remark this answers is not in the document]
Comment (piekczyk) [reply components/history/PositionHistory.tsx:1]: done

File: features/ajna/positions/common/controls/AjnaPositionHistoryController.tsx [hunk 2]
```diff
@@ -11,10 +14,26 @@ export function AjnaPositionViewInfoPlaceholder() {
   )
 }
 
-export function AjnaPositionViewHistoryPlaceholder() {
+export function AjnaPositionHistoryController() {
```
[the remark this answers is not in the document]
Comment (piekczyk) [reply features/ajna/positions/common/controls/AjnaPositionHistoryController.tsx:17]: the same controller will be used for multiply as multiply contains borrow events as well

File: components/history/PositionHistory.tsx (hunk 1 above)
Comment (piekczyk) [reply components/history/PositionHistory.tsx]: done

**Review by piotrkonowrocki — APPROVED** [review]

**Pull request merged**